apps/s_time.c

```diff
@@ -176,8 +175,9 @@ int s_time_main(int argc, char **argv)
             break;
         case OPT_WWW:
             www_path = opt_arg();
-            if (strlen(www_path) > MYBUFSIZ - 100) {
-                BIO_printf(bio_err, "%s: -www option too long\n", prog);
+            buf_len = strlen(www_path) + sizeof(fmt_http_get_cmd) - 2;  /* 2 is for %s */
+            if (buf_len > (int)sizeof(buf)) {
```

Comment:
Wouldn't it be better to use a `size_t` variable here rather than reuse buf_len? i.e. strlen returns a size_t but we (implicitly) cast it to int, and then we compare it to the result of a sizeof which also returns a size_t, but we (explicitly) cast it to an int!!!